docs/Installation.md

```diff
@@ -23,6 +23,10 @@ Consequently, to install and use the ML-Agents Toolkit you will need to:
 - Install Unity (2018.4 or later)
 - Install Python (3.6.1 or higher)
 - Clone this repository (Optional)
+  - Please note that if you do not clone the repository, then you will not be 
```

Comment:
I think our "Notes" are mostly formatted with 
`__Note:__ If you do not clone the ...`

Reply:
fixdd